packages/core/src/lib/Slider/types.ts

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+export interface ISliderProps {
+    /**
+     * pass id for html input component
+     */
+    rangeId: string;
+    /**
+     * pass any Label for Slider
+     */
+    rangeLabel?: string;
+    /**
+     * pass maximum Range for Slider
+     */
+    rangeMax: number;
+    /**
+     * pass minimum Range for Slider
+     */
+    rangeMin: number;
+    /**
+     * Pass for setting current value 
+     */
+    rangeValue?: number;
+    /**
+     * Pass for setting Changed event 
+     */
+    onChanges?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    /**
```

Comment:
let use `onChange` as we use the same in all the components 

In comments you can write `callback to onchange event`